src/main/java/org/apache/cassandra/sidecar/utils/SSTableImporter.java

```diff
@@ -623,4 +645,38 @@ public ImportOptions build()
             }
         }
     }
+
+    /**
+     * Key used for the map of queues
+     */
+    protected static class ImportKey
```

Comment:
1. Maybe call it `ImportId`?
2. Can change the access modifier to `private`? I think it is unexpected to create its subclasses. `protected` adds maintenance burden